src/main/clojure/clara/rules/dsl.clj

```diff
@@ -135,7 +135,12 @@
            (parse-expression nested-expr expr-meta)))
 
    (contains? #{'test :test} (first expression))
-   {:constraints (vec (rest expression))}
+   (if (= 1 (count expression))
+     (throw-dsl-ex (str "Empty test conditions are not allowed.")
```

Comment:
I think this might be clearer with a colon in front of the "test" to denote that it is a keyword, i.e. 

````
"Empty :test conditions are not allowed."
````

I realize that the symbol "test" is allowed as well, but I suspect that :test is the more common usage since that is what the [docs](http://www.clara-rules.org/docs/expressions/#tests) use.

Thoughts?

Reply:
Ya good point